governance/xc_admin/packages/xc_admin_frontend/next.config.js

```diff
@@ -5,6 +5,7 @@ const nextConfig = {
     externalDir: true,
   },
   webpack(config) {
+    config.experiments = { asyncWebAssembly: true }
```

Comment:
here and below is fixing some weird build issues from pulling the xc-governance-sdk into xc-admin & the frontend components.